processing/app/deps.py

```diff
@@ -1,12 +1,54 @@
-from sqlmodel import create_engine
 from app.config import Settings
+from app.repository.survey_repository import SurveyRepository
+from app.repository.task_repository import TaskRepository
+from app.services.analysis_service import AnalysisService
+from app.services.survey_service import SurveyService
+from app.services.task_service import TaskService
 from ollama import AsyncClient
+from sqlmodel import create_engine
+from sqlmodel import Session
+
 
 def get_database_engine():
     postgres_url = str(Settings().database.db_connection_url)
-    engine = create_engine(postgres_url, echo=True, future=True)
+    engine = create_engine(postgres_url)
```

Comment:
To avoid having to refresh objects after the commit, you lets pass `expire_on_commit=False` flag. You can read more about it here: https://docs.sqlalchemy.org/en/20/orm/session_api.html#sqlalchemy.orm.Session.params.expire_on_commit